app/graphql/types/base_object.rb

```diff
@@ -17,6 +17,21 @@ def model_class(new_model_class = nil)
           @model_class ||= "::#{to_s.demodulize}".safe_constantize
         end
       end
+
+      def authorized?(object, context)
+        permission = object.class.instance_variable_get(:@rbac_permission)
+        # Warn the developer if there is a missing RBAC permission for a queried GQL type
+        if permission.nil?
+          Rails.logger.warn("There is no RBAC enforced on to the #{object.class} GraphQL type!")
+          return true # FIXME: should not return true to cover new added objects
```

Comment:
It should return `true` but maybe QE wants to catch these warnings later if they can. But we should maybe add a way to lift this warning in certain cases. Probably with a `skip_rbac!` method...